sunpy/coordinates/tests/test_utils.py

```diff
@@ -2,12 +2,13 @@
 import pytest
 
 import astropy.units as u
-from astropy.coordinates import ConvertError, SkyCoord
+from astropy.coordinates import ConvertError, SkyCoord, get_body
+from astropy.time import Time
```

Comment:
```suggestion
from astropy.coordinates import ConvertError, SkyCoord
from astropy.tests.helper import assert_quantity_allclose
```
Stop using `get_body()`. =)